app/components/Application/ApplicationDecision.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Alert} from 'react-bootstrap';
+
+interface Props {
+  actionRequired: Boolean;
+  decision: string;
+  reviewComments: string[];
+}
+
+const BS_ALERT = {
+  REQUESTED_CHANGES: 'warning',
+  REJECTED: 'danger',
+  APPROVED: 'success'
+};
+
+const DECISION_TEXT = {
+  REQUESTED_CHANGES: '',
+  REJECTED:
+    'Thank you for applying to the CIIP. Unfortunately this submission did not meet program requirements and a payment cannot be granted.',
+  APPROVED:
+    'This application has been approved. Your incentive payment is being processed. No further action is required from you at this time.'
+};
+
+const DECISION_PRETTY = {
+  REQUESTED_CHANGES: 'Changes Requested',
+  REJECTED: 'Rejected',
+  APPROVED: 'Approved'
+};
+
+const ApplicationDecision: React.FunctionComponent<Props> = ({
+  actionRequired,
+  decision,
+  reviewComments,
+  children
+}) => {
+  const heading = actionRequired
+    ? 'Action Required'
+    : decision === 'REQUESTED_CHANGES'
+    ? 'Changes Requested'
+    : `Decision: ${DECISION_PRETTY[decision]}`;
+
+  // eslint-disable-next-line react/no-array-index-key
```

Comment:
It is okay to disable it here because the comments array will not be updated while this component is mounted. Can you add a comment to justify disabling the linter?